content/ja/cloud-computing.md

```diff
@@ -0,0 +1,18 @@
+---
+title: クラウドコンピューティング
+status: Completed
+category: concept
+tags: ["infrastructure", "fundamental", ""]
```

Comment:
It is recommended to localize each tag string like:

```suggestion
tags: ["インフラストラクチャー", "基本概念", ""]
```

(This might not be the correct example. I am totally not good at Japanese! 😆)

Reply:
Thank you for review! Indeed, it is good to add localized tag in content. I will add localized tags.